anchore_engine/services/policy_engine/engine/feeds/feeds.py

```diff
@@ -1382,7 +1245,7 @@ def record_count(self, group_name, db):
                 .filter(Vulnerability.namespace_name == group_name)
                 .count()
             )
-        except Exception as e:
+        except Exception:
```

Comment:
Same comment as above, would you want to log the message from the exception here?

Reply:
same as before, gets logged by magic